test/suite/test_import11.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# test_import11.py
+#    Tests import of tiered tables using backup:export cursor and metadata_file import option.
+#
+
+import glob, os, random, re, shutil, string
+import wttest
+from helper_tiered import TieredConfigMixin, tiered_storage_sources
+from wtscenario import make_scenarios
+
+# Shared base class used by import tests.
+class test_import_base(TieredConfigMixin, wttest.WiredTigerTestCase):
+
+    # Insert or update a key/value at the supplied timestamp.
+    def update(self, uri, key, value, ts):
+        cursor = self.session.open_cursor(uri)
+        self.session.begin_transaction()
+        if type(value) in [list, tuple]:
+            cursor.set_key(key)
+            cursor.set_value(*value)
+            cursor.insert()
```

Comment:
Does the `cursor[key] = value` not work in this case?  If it doesn't, we should file a ticket to see if we can fix it.

Reply:
It works. I just copied the method from another test. I've removed unnecessary code. Thanks!